core/src/components/title/title.ios.scss

```diff
@@ -31,7 +31,7 @@
 
   position: relative;
 
-  font-size: 13px;
+  font-size: dynamic-font-min(1, 13px);
```

Comment:
This is going to set a minimum font size. Did you mean to use `dynamic-font-max` instead?